lib/config.js

```diff
@@ -38,6 +38,31 @@ const getLocalConfig = ({ file, dir = process.cwd() }) => {
   return result && _.isPlainObject(result.config) ? result.config : localConfig;
 };
 
+const fetchConfigurationFromGitHub = async pattern => {
+  const regex = /^github>([^/]+)\/([^#:]+)(?::([^#]+))?(?:#(.+))?$/;
+  const match = pattern.match(regex);
+
+  if (!match) {
+    throw new Error('Invalid Extended Configuration from GitHub');
```

Comment:
This could perhaps be improved, maybe add `github` as `protocol`/(named) match (like `owner` etc. below). Also please look for other instances of `throw e()`.

Reply:
Honestly, I have never been a big fan of regex, I think I prefer JSONana to extract information, and build logical around it.

I will update the error following other patterns!

Thanks!

Reply:
Maybe we could parse it as a `new URL()`